### ISSUE TITLE
Helper text and placeholder text have insufficient color contrast ratio (less than 4.5:1)

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4630]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
- Test URL: https://clarity.design/documentation
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Open the test url.
2. Go to any one of the clarity components which has helper text provided and observe the color contrast.

**Expected result:**
Color contrast of helper text should be at least 4.5:1 so that low vision users can see the helper text with better contrast.

**Actual result:**
The contrast ratio is: 3.22:1, so low vision users might find it difficult to read the helper text.

**Comment by gkhadK** [comment]
This issue is impacting all the helper text provided for the clarity components. Similar issue is observed for helper text.

**Comment by chlane** [comment]
https://github.com/vmware/clarity/issues/3890 is a related issue

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
